src/main/java/StringManipulation.java

```diff
@@ -1,32 +1,128 @@
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+/**
+ * The implementation class StringManipulation for the StringManipulationInterface.
+ *
+ * @author DeWayne Dantzler dewayne.dantler@bellevuecollege.edu
+ */
 public class StringManipulation implements StringManipulationInterface {
-
+	private String _MyString;
+	
+	public StringManipulation(){
+		_MyString = "";
+	}
+	
     @Override
     public String getString() {
-        return null;
+        return _MyString;
     }
 
     @Override
     public void setString(String string) {
+    	_MyString = string;
     }
 
     @Override
     public int count() {
-        return 0;
+    	int count = 0;
+    	if(_MyString != null && !_MyString.isEmpty()) {
+    		List<String> result = new Scanner(_MyString)
+    			     .useDelimiter(" ")
+    			     .tokens()
+    			     .collect(Collectors.toList());
+    		count = result.size();
+    	}
+    	
```

Comment:
No, spaces do not cause an issue because whitespace is used as delimiter and removed from the String when added to the collection. This is my understanding.